params/bootstrap.go

```diff
@@ -9,17 +9,17 @@ import (
 var log = logging.Logger("params")
 
 // BootstrappersInfosFor returns address information of bootstrap peers for a given network.
-func BootstrappersInfosFor(net Network) ([]peer.AddrInfo, error) {
-	bs, err := BootstrappersFor(net)
+func BootstrappersInfosFor(net Network) (Bootstrappers, error) {
+	bs, err := bootstrappersFor(net)
 	if err != nil {
 		return nil, err
 	}
 
 	return parseAddrInfos(bs)
 }
 
-// BootstrappersFor reports multiaddresses of bootstrap peers for a given network.
-func BootstrappersFor(net Network) ([]string, error) {
+// bootstrappersFor reports multiaddresses of bootstrap peers for a given network.
+func bootstrappersFor(net Network) ([]string, error) {
```

Comment:
If me make this private, we can omit Infos from BootstrappersInfoFor to make shorter. Also, BootstrappersInfoFor returns the type Bootstrappers and not the info for the type bootstrappers